frontend/public/kubevirt/components/nic.jsx

```diff
@@ -85,8 +85,9 @@ const NIC_TYPE_VM = 'nic-type-vm';
 const NIC_TYPE_CREATE = 'nic-type-create';
 
 export const NicRow = (onChange, onAccept, onCancel) => ({obj: nic}) => {
+  const namespace = (nic.vm && nic.vm.metadata.namespace) || (nic.vmTemplate && nic.vmTemplate.metadata.namespace);
```

Comment:
you can use `getNamespace` from `web-ui-components`  - `getNamespace(nic.vm) || getNamespace(nic.vmTemplate)`

Reply:
Wow, thanks :+1: